src/autolabel/tasks/attribute_extraction.py

```diff
@@ -118,6 +139,8 @@ def construct_prompt(
         for eg in examples:
```

Comment:
Just confirming - this effectively means that if we had 5 examples passed in and 1 one of them doesn't have all GT labels populated, we'll now only send 4 examples in? We don't explicitly go and fetch a new example to ensure that 5 are going in, correct? 
I think this behavior is fine as is, wanted to make sure I am understanding this correctly.

Reply:
Yeah, we do plan to revamp example selection/vector search to improve on this behavior, but that will come as a followup.